src/main/java/nl/mpcjanssen/simpletask/util/Util.kt

```diff
@@ -188,6 +189,11 @@ fun join(s: Collection<String>?, delimiter: String): String {
     return s.joinToString(delimiter)
 }
 
+fun getColorDrawable(ctx: Context, attr: Int): Drawable {
+    val color = ctx.obtainStyledAttributes(intArrayOf(attr))
+    return ContextCompat.getDrawable(ctx, color.getResourceId(0, 0))
+}
+
```

Comment:
Right now it's only called from Simpletask.kt, but my gut told me to put it in Util. It'll be nice if we ever change color anywhere else (eg, widgets, the AddTask screen). Or if we finally break up Simpletask.kt into smaller files like we really should.